### PR TITLE
Introduce logs drain as a beta feature

### LINKED ISSUE #356: Add page about the log drain feature
With a warning at the beginning stating it's a beta feature

### PR DESCRIPTION
**Pull request by EtienneM**
https://documentation-service-pr370.scalingo.io/platform/app/log-drain

@Soulou Do you want to enforce an environment variable name for the token, to prepare the release of the non-beta feature? Or it will not be about an environment variable name?

Fix #356

### CODE REVIEW

**Comment by Soulou** [comment]
For me it should not be in the environment, too clumsy. It will be a custom integration where the user put the URL/token or as an Addon. (I'm in favor of the first)

**Comment by Soulou** [comment]
There is nothing about syslog, it should be present (UDP/TCP/TCP+TLS)

File: _posts/platform/app/2000-01-01-log-drain.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: Add a Log Drain to your App
+nav: Log Drain
+modified_at: 2018-06-15 10:00:00
+tags: Papertrail Logentries Logmatic Graylog logs integration
```
Comment (yannski) [inline _posts/platform/app/2000-01-01-log-drain.md]: tags are not used ATM. Theses names should appear explicitely in the "real" content of the page.

**Comment by EtienneM** [comment]
> There is nothing about syslog, it should be present (UDP/TCP/TCP+TLS)

I don't really get what you mean here

**Comment by Soulou** [comment]
We don't support only private providers to drain logs to, but also standard protocol : syslog.

Syslog over yep/TCP/TCP+TLS

**Pull request merged**